src/data/finset/basic.lean

```diff
@@ -758,6 +794,37 @@ theorem inter_subset_right (s₁ s₂ : finset α) : s₁ ∩ s₂ ⊆ s₂ := 
 theorem subset_inter {s₁ s₂ s₃ : finset α} : s₁ ⊆ s₂ → s₁ ⊆ s₃ → s₁ ⊆ s₂ ∩ s₃ :=
 by simp only [subset_iff, mem_inter] {contextual:=tt}; intros; split; trivial
 
+lemma subset_inter_iff {s₁ s₂ s₃ : finset α} :
+  s₁ ⊆ s₂ ∩ s₃ ↔ s₁ ⊆ s₂ ∧ s₁ ⊆ s₃ :=
+⟨λ h, ⟨λ x hx, mem_of_mem_inter_left (h hx), λ x hx, mem_of_mem_inter_right (h hx)⟩, λ ⟨h₁, h₂⟩,
+  subset_inter h₁ h₂⟩
+
+lemma inter_eq_left_iff {s₁ s₂ : finset α} :
+  s₁ ∩ s₂ = s₁ ↔ s₁ ⊆ s₂ :=
```

Comment:
This lemma and the below are already present around line 945, called `inter_eq_left_iff_subset`.